satpy/etc/readers/mersi_ll_l1b.yaml

```diff
@@ -0,0 +1,251 @@
+reader:
+  description: FY-3D Medium Resolution Spectral Imager 2 (MERSI-2) L1B Reader
+  name: mersi_ll_l1b
+  sensors: [mersi-ll]
```

Comment:
Should this be `mersi-ll` or `mersi-2-ll` or `mersi2-ll`? The `2` is important isn't it?

Reply:
No, they are totally separate instruments and I think the names are correct. FY-3D and FY-3F fly the `MERSI-2` instrument while FY-3E flies `MERSI-LL`

That said, your comment has made me realise I forgot to change the description. It's the copy-pasted one from the mersi-2 yaml. Will fix that later today!

Reply:
Ah gotcha. Sounds good then.